### PR TITLE
Please add Gnu IceCat

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/123131)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Exception during executable launch operation System.AggregateException: One or more errors occurred. (The specified executable is not a valid application for this OS platform.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
image

File: manifests/i/Icecat/Icecat/115.3.1esr/Icecat.Icecat.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+PackageIdentifier: Icecat.Icecat
+PackageVersion: "115.3.1esr"
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
```
Comment (stephengillie) [inline manifests/i/Icecat/Icecat/115.3.1esr/Icecat.Icecat.installer.yaml:13]:
```suggestion
UpgradeBehavior: install
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Comment by alithechemist** [comment]
ok i'm missing vc redist...
thanks. I'll se how to pack everything together.

File: manifests/i/Icecat/Icecat/115.3.1esr/Icecat.Icecat.installer.yaml (hunk 1 above)
Comment (alithechemist) [reply manifests/i/Icecat/Icecat/115.3.1esr/Icecat.Icecat.installer.yaml:13]: thanks!

**Comment by alithechemist** [comment]
I just tested again the command with the silent switch on a windows 10 vm on a normal command prompt, and it just works. 
Only thing is the warning for unknown publisher. 
Is it that the reason why the package cannot be installed without user input?

Thanks

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

Dependencies aren't quite working in the Automatic Pipeline.

**Comment by stephengillie** [comment]
Hi @alithechemist,

This is an unusual situation, where "The IceCat project does not currently distribute binaries...". Our process is built on using the ISV's official hosting, but here there is none by definition. So we'll have to review, sorry for the delay.

**Comment by alithechemist** [comment]
Hi, thanks, for letting me know. I know Icecat is a bit weird for this, but I've been talking to the Gnu people and it seemed ok to them that i distribute binaries, only thing they can't endorse this for systems like Windows, Android oror Mac; reason why I didn't do g/GNU/Icecat and i changed the publisher url.

Anyway, thank you. I believe i just need to wait for you guys to look into this.
Thanks,
Chippy.

**Comment by alithechemist** [comment]
Hi, is there any news on this?
There is a new version as well to add.

Thanks.
Chip

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/i/Icecat/Icecat/115.4.0esr/Icecat.Icecat.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  SilentWithProgress: /S
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```
Comment (stephengillie) [inline manifests/i/Icecat/Icecat/115.4.0esr/Icecat.Icecat.installer.yaml:19]:
```suggestion
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**